ldap/admin/src/logconv.py

```diff
@@ -619,13 +619,13 @@ def _match_line(self, line: str, bytes_read: int):
                 if parse_start and parse_stop:
                     if parse_start.microsecond or parse_stop.microsecond:
                         if not parse_start <= norm_timestamp <= parse_stop:
-                            self.logger.error(f"Timestamp {norm_timestamp} outside of range ({parse_start} - {parse_stop})")
-                            return False
+                            self.logger.debug(f"Timestamp {norm_timestamp} outside of range ({parse_start} - {parse_stop})")
+                            return
```

Comment:
_return_ seems a bit suspicious as __match_line_ function is supposed to return a boolean 
 as the time does not change in the _pattern,action_ loop maybe return False was ok.
 BTW __match_line_ function should also return False if the _pattern,action_ loops ends without any matches
 (i.e a final return False is missing in the __match_line_ function)